### PR TITLE
Increase default root disk size

### PR DESCRIPTION
**Pull request by atsikham**

### CODE REVIEW

File: CHANGELOG/0.0.1.md [hunk 1]
```diff
@@ -4,4 +4,5 @@
 
 ### Added
 
-* [#1637](https://github.com/epiphany-platform/epiphany/issues/1637) First module implementation. It can create VPC, subnet, internet gateway, security group, route table and instances (in that single subnet).  
+* [#1637](https://github.com/epiphany-platform/epiphany/issues/1637) - First module implementation. It can create VPC, subnet, internet gateway, security group, route table and instances (in that single subnet)
+* [#21](https://github.com/epiphany-platform/m-aws-basic-infrastructure/issues/21) - Add the ability to extend root disk size
```
Comment (mkyc) [inline CHANGELOG/0.0.1.md:8]: [0.0.1 is already released](https://github.com/epiphany-platform/m-aws-basic-infrastructure/releases/tag/v0.0.1), please move it to another file named 0.0.2

File: resources/terraform/modules/ec2/main.tf [hunk 2]
```diff
@@ -163,6 +163,10 @@ resource "aws_instance" "awsbi" {
   associate_public_ip_address = var.use_public_ip
   key_name                    = var.key_name
 
+  root_block_device {
+    volume_size = var.root_disk_size
```
Comment (mkyc) [inline resources/terraform/modules/ec2/main.tf:167]:
```suggestion
    volume_size = var.root_volume_size
```

File: resources/terraform/main.tf [hunk 3]
```diff
@@ -8,6 +8,7 @@ module "ec2" {
 
   name              = var.name
   instance_count    = var.instance_count
+  root_disk_size    = var.root_disk_size
```
Comment (mkyc) [inline resources/terraform/main.tf:11]:
```suggestion
  root_volume_size    = var.root_volume_size
```

File: resources/terraform/modules/ec2/variables.tf [hunk 4]
```diff
@@ -20,6 +20,11 @@ variable "instance_count" {
   type        = number
 }
 
+variable "root_disk_size" {
+  description = "The size of the volume in gibibytes (GiB)"
```
Comment (mkyc) [inline resources/terraform/modules/ec2/variables.tf:24]:
```suggestion
variable "root_volume_size" {
  description = "The size of the root volume in gibibytes (GiB)"
```

File: resources/terraform/variables.tf [hunk 5]
```diff
@@ -8,6 +8,12 @@ variable "instance_count" {
   type        = number
 }
 
+variable "root_disk_size" {
+  description = "The size of the volume in gibibytes (GiB)"
```
Comment (mkyc) [inline resources/terraform/variables.tf:12]:
```suggestion
variable "root_volume_size" {
  description = "The size of the root volume in gibibytes (GiB)"
```

**Review by mkyc — CHANGES_REQUESTED** [review]

**Review by erzetpe — APPROVED** [review]

**Review by mkyc — APPROVED** [review]
LGTM

**Pull request merged**